mavis/tools.py

```diff
@@ -478,15 +478,22 @@ def _convert_tool_output(input_file, file_type=SUPPORTED_TOOL.MAVIS, stranded=Fa
                 'normalized_RD',
                 'e-val1', 'e-val2', 'e-val3', 'e-val4', 'q0'
             ])
-    elif file_type in [SUPPORTED_TOOL.DELLY, SUPPORTED_TOOL.MANTA, SUPPORTED_TOOL.PINDEL, SUPPORTED_TOOL.VCF, SUPPORTED_TOOL.BREAKSEQ]:
+    elif file_type in [SUPPORTED_TOOL.DELLY, SUPPORTED_TOOL.MANTA, SUPPORTED_TOOL.PINDEL, SUPPORTED_TOOL.VCF, SUPPORTED_TOOL.BREAKSEQ, SUPPORTED_TOOL.STRELKA]:
         rows = []
         vfile = VariantFile(input_file)
         try:
             vfile.header.info.add('END', number=1, type='Integer', description='End of the interval')
         except ValueError:
             pass
         for vcf_record in vfile.fetch():
```

Comment:
how are you filtering out the SNVs?

Reply:
I assumed we would only use the indel from Strelka as input...

Reply:
ya, if the splitting of indels and snvs is done by strelka then you can ignore my SNV related comments. If it is not done by default though then we will need to worry about verbosely filtering them